src/custom/api/gnosisProtocol/errors/OperatorError.ts

```diff
@@ -34,6 +35,7 @@ export enum ApiErrorCodeDetails {
   InsufficientValidTo = 'The order you are signing is already expired. This can happen if you set a short expiration in the settings and waited too long before signing the transaction. Please try again.',
   InsufficientFunds = "The account doesn't have enough funds",
   UnsupportedToken = 'One of the tokens you are trading is unsupported. Please read the FAQ for more info.',
+  SellAmountDoesNotCoverFee = 'The sell amount for the sell order is lower than the fee.',
```

Comment:
Can you verify all our errors are in sync since you are touching this. I believe at least the NotEnoughLiquidy error  was added in the backend but not in the frontend

Reply:
#1896